node/cn/handler.go

```diff
@@ -1051,26 +1051,24 @@ func (pm *ProtocolManager) BroadcastBlockHash(block *types.Block) {
 		"recipients", len(peersWithoutBlock), "duration", common.PrettyDuration(time.Since(block.ReceivedAt)))
 }
 
-// BroadcastTxs will propagate a batch of transactions to all peers which are not known to
+// BroadcastTxs propagates a batch of transactions to its peers which are not known to
```

Comment:
```suggestion
// BroadcastTxs propagates a batch of transactions to their peers which are not known to
```

Reply:
Why "their" peers? Since "a node" sends transactions to "its" peers, "its" seems to be more appropriate. How do you think? "Their peers" can mislead the readers in that their might be a `m to n` or `n to 1` peer relationship.